.gitignore

```diff
@@ -14,3 +14,4 @@ patron_auths.json
 push.bat
 secondaries.txt
 venv.bat
+start.sh
```

Comment:
This could be voided, was used in debugging and testing but it is no longer needed.